tests/AnnotatedRepositoryFactoryTest.php

```diff
@@ -27,9 +27,9 @@ public function setUp(): void
     {
         parent::setUp();
```

Comment:
You can remove the call to `parent::setUp();,` because it does nothing.
The `setUp` method in `TestCase` is there only to make sure that the method is callable before each test, even if a test does not implement it.